.travis.yml

```diff
@@ -37,7 +37,7 @@ addons:
     project:
       name: "fontforge/fontforge"
       description: "Free (libre) font editor for Windows, Mac OS X and GNU+Linux"
-    build_command_prepend: cov-configure --comptype gcc --compiler $CC --template && ./bootstrap --force && ./configure $FFCONFIG
+    build_command_prepend: cov-configure --comptype gcc --compiler $CC --template && ./bootstrap --force && ./configure $FFCONFIG --enable-fontforge-extras
```

Comment:
Why not just move that flag into FFCONFIG

Reply:
I wasn't sure about if it should be default or not.

Do you think that maybe this should be default build, disable to not include extras?

Reply:
No, definitely this should remain be enable to include. 

What I mean is that this stuff used to get tested under CI, which is fine. But what you did here was to basically only run it when we do coverage builds, which is very infrequent.

Reply:
Ok,
How do you suggest it be integrated in configure.ac and Makefile.am ?
the distcheck has a lot of stuff disabled, commented out. these should be checked too.
How can we test with this as default off, but also enabled? Add another test after linux-nox ?

Reply:
Don't do anything there. Just add it to FFCONFIG, as suggested before: https://github.com/fontforge/fontforge/blob/master/.travis.yml#L61

That will cause that option to be enabled on all the Linux and Mac CI builds.

Reply:
Checked enabled on in #3284, came up all green.
FFCONFIG tests on on all builds.
...but we're looping back to the same question? How can you also test with it off?

Reply:
Why do we need to test both configurations? If you've seen it work without the option enabled then there's not much that can go wrong, it's not like the build system is in a constant state of flux.

It's good enough to just enable it in ci so that any obvious build issues with the code itself are not missed.

Reply:
Sorry, but it's just not practical to test every build option under the sun in ci. As mentioned before, I'm more  concerned about catching any regressions on the code, and less so with the build system. If you're making changes to that, then I'd expect you'd test any build config changes to see if they work. Adding a build just to see if it builds properly without contrib code adds very little value while just blowing out the ci build time. 


As for your other build system changes, I'm not sure how it's related to gdk, considering that all ci builds at the moment still build without it too.

Reply:
All green checkmarks on #3286, and #3284.
"make dist" copied all the dist files with or without extras enabled.
wiki on glib and gdk shows dependencies...so for now, I'll try things that don't have gdk, harfbuzz, cairo, pango, X.
different topic, but related to this last comment... #3290 built ok locally, but came-out as "X" on fontforge pull request, so that needs more attention as to why it broke.